custom_components/opensprinkler/__init__.py

```diff
@@ -200,13 +200,12 @@ def _get_controller_attributes(self, controller):
             "last_reboot_time",
         ]:
             iso_attr = attr + "_iso"
-            if getattr(controller, attr) == 0:
+            timestamp = getattr(controller, attr)
+            if not timestamp:
```

Comment:
this accounts for a new controller that has never been run so timestamp is `None` as well as 0